### PR TITLE
[PDI-19732] - Transformation connecting to S3 fails when using URL s3n://s3n/{s3_bucket_name}

### PR DESCRIPTION
**Pull request by bcostahitachivantara**

### CODE REVIEW

**Comment by buildguy** [comment]
### :x: Build failed in 2m 8s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl s3-vfs
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :x: Build failed in 2m 19s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl s3-vfs
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :x: Build failed in 2m 17s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl s3-vfs
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

File: s3-vfs/src/main/java/org/pentaho/amazon/s3/S3FileOutputDialog.java [hunk 1]
```diff
@@ -1158,7 +1158,7 @@ public void widgetSelected( SelectionEvent e ) {
               BaseMessages.getString( BASE_PKG, "System.FileType.AllFiles" ) };
 
           if ( StringUtils.isEmpty( wFilename.getText().trim() ) ) {
-            wFilename.setText( "s3n://s3n/" );
+            wFilename.setText( "s3a://" );
```
Comment (smmribeiro) [inline s3-vfs/src/main/java/org/pentaho/amazon/s3/S3FileOutputDialog.java:1161]: Is it really "s3a" or should it stay as "s3n" like on lines 323 and 1478?

Comment (bcostahitachivantara) [reply s3-vfs/src/main/java/org/pentaho/amazon/s3/S3FileOutputDialog.java:1161]: yes, the s3a is the new default format

**Comment by buildguy** [comment]
### :x: Build failed in 2m 29s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl s3-vfs
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :x: Build failed in 2m 14s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl s3-vfs
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Review by smmribeiro — APPROVED** [review]
Looks good!

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 10m 11s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl s3-vfs
```
---
:ok_hand: All tests passed!
 
Tests run: **147**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/big-data-plugin/job/PR-2468/6/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**